docs/_docs/starting-nodes.adoc

```diff
@@ -213,6 +213,28 @@ You can also pass JVM options by using the `-J` prefix:
 ----
 ./ignite.sh -J-Xmx6G -J-DIGNITE_TO_STRING_INCLUDE_SENSITIVE=false
 ----
+
+== Ignite Internal Settings
+Ignite provide more then hundred internal flags to tweak som apects of the node behavior.
+You can see all flags, their descriptions and default values with the following command:
+
+[source, shell]
+----
+./ignite.sh -systemProps
```

Comment:
Please add a tab for Windows.

Reply:
We don't have windows examples for other commands on this page.
I think we should keep it as is.